### PR TITLE
Participation frais logement

### PR DESCRIPTION
**Pull request by fpagnoux**
Connected to sgmap/openfisca-france#30

### CODE REVIEW

**Comment by MattiSG** [comment]
Needs migration of tests.

**Comment by fpagnoux** [comment]
Here you are

File: migrations/gratuit-heberge.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+var mongoose = require('mongoose');
```
Comment (MattiSG) [inline migrations/gratuit-heberge.js]: To improve discoverability, use version numbers as file names: `v<origin>_v<destination>`.

**Comment by MattiSG** [comment]
Without migration: 
capture d ecran 2015-11-18 a 18 08 12
After migrating: 
capture d ecran 2015-11-18 a 18 18 09
…which is the same as in production right now. Seems thus ok, but means there is no test on the topic atm.

**Comment by MattiSG** [comment]
GTM

**Pull request merged**